crates/holochain/src/core/workflow/sys_validation_workflow/validate_op_tests.rs

```diff
@@ -232,99 +252,24 @@ async fn validate_create_op_with_prev_from_network() {
     create_action.timestamp = Timestamp::now().into();
     let op = DhtOp::RegisterAgentActivity(fixt!(Signature), Action::Create(create_action));
 
-    test_case
-        .cascade_mut()
-        .expect_retrieve_action()
-        .times(1)
-        .returning({
-            let previous_action = previous_action.clone();
-            move |_, _| {
-                let previous_action = previous_action.clone();
-                async move { Ok(Some((previous_action, CascadeSource::Local))) }.boxed()
-            }
-        });
-
     test_case
         .cascade_mut()
         .expect_retrieve()
         .times(1)
-        .returning(move |_hash, _options| {
-            let previous_action = previous_action.clone();
-            async move {
-                Ok(Some((
-                    Record::new(previous_action, None),
-                    CascadeSource::Network,
-                )))
-            }
-            .boxed()
-        });
+        .returning(move |_, _| async move { Ok(None) }.boxed());
 
-    let outcome = test_case
-        .with_incoming_ops_sender()
-        .with_op(op)
-        .execute()
-        .await
-        .unwrap();
+    let outcome = test_case.with_op(op).run().await.unwrap();
 
-    assert!(
-        matches!(outcome, Outcome::Accepted),
-        "Expected Accepted but actual outcome was {:?}",
-        outcome
-    );
-}
-
-// TODO This should not error but also represents a missed opportunity to capture an op.
-//      At the moment this is silently ignored because the `incoming_dht_ops_sender` is optional.
-#[tokio::test(flavor = "multi_thread")]
-async fn validate_create_op_with_prev_from_network_but_missing_op_sender() {
```

Comment:
No longer possible, it's a required input to the workflow. It's not passed to validate_op at all so it's not relevant to test here